src/js/util.js

```diff
@@ -17,5 +17,69 @@ define(function (require, exports) {
   exports.track = function(event, data) {
     Intercom('trackEvent', event, data);
   };
+  
+  // Take an array of URLs and return a template of the form
+  // `//{s}.foo.bar/tiles/baz` and an array of substitutions for `s` that will
+  // generate the original URLs
+  exports.templatizeURLs = function templatizeURLs(urls) {
```

Comment:
Yah, Leaflet supports a templated URL for a TileLayer. The UTFGrid plugin takes advantage of the same mechanism for the UTFGrid URL. But TileJSON provides multiple URLs as an array, rather than a template and an array of substitutions. The TileJSON plugin punts on that issue and just uses the first URL of the array. Elsewhere, when we use the TileJSON data directly and not through the plugin, we do the same thing. Since we're not really using any other features of the TileJSON plugin/data, the simplest approach is to create the TileLayer directly and use a utility function to convert from an array to a template + substitution array. We could incorporate that functionality into the TileJSON plugin, but without grid support, it still wouldn't quite suffice.